crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -348,21 +385,27 @@ module deepbook::clob_v2 {
         )
     }
 
+    /// Deposit base asset into the pool.
+    ///
+    /// Aborts if the `Coin` is zero.
+    /// Emits: `DepositAsset<BaseAsset>` event.
     public fun deposit_base<BaseAsset, QuoteAsset>(
         pool: &mut Pool<BaseAsset, QuoteAsset>,
         coin: Coin<BaseAsset>,
         account_cap: &AccountCap
     ) {
-        let quantity = coin::value(&coin);
-        assert!(quantity != 0, EInsufficientBaseCoin);
+        let coin_amount = coin::value(&coin);
+        assert!(coin_amount != 0, EInsufficientBaseCoin);
+
         custodian::increase_user_available_balance(
             &mut pool.base_custodian,
             account_owner(account_cap),
             coin::into_balance(coin)
         );
+
         event::emit(DepositAsset<BaseAsset>{
             pool_id: *object::uid_as_inner(&pool.id),
-            quantity,
+            quantity: coin_amount,
```

Comment:
Not sure if this change improves the code... field name punning makes the code shorter and easier to read

Reply:
But then it's easily confused with `quantity` that is used everywhere else.